drydock/plugin.py

```diff
@@ -174,10 +174,14 @@ def get_sync_waves_for_resource(resource_name: str) -> SYNC_WAVES_ORDER_ATTRS_TY
 
 tutor_hooks.Filters.CONFIG_DEFAULTS.add_items([("OPENEDX_DEBUG_COOKIE", "ednx_enable_debug")])
 tutor_hooks.Filters.CONFIG_OVERRIDES.add_items([
+tutor_hooks.Filters.CONFIG_OVERRIDES.add_items(
+    [
         # This values are not prefixed with DRYDOCK_
+        ("FORUM_MONGODB_DATABASE", "cs_comments_service"),
```

Comment:
This is because we are no longer using the Tutor Forum plugin right?

Reply:
No, the plugin is still used add some settings to the edxapp services. But since v19 it's [hardcoding the value](https://github.com/overhangio/tutor-forum/blob/5d3d2dd7d1ea3c9a311ac5619eb5931a111ba363/tutorforum/plugin.py#L29) instead of using the plugin variable. That line should be something like `FORUM_MONGODB_DATABASE = "{{ FORUM_MONGODB_DATABASE }}"`